src/device-registry/controllers/create-airqloud.js

```diff
@@ -71,10 +71,10 @@ const createAirqloud = {
       const hasErrors = !validationResult(req).isEmpty();
       if (hasErrors) {
         let nestedErrors = validationResult(req).errors[0].nestedErrors;
-        return badRequest(
+        return errors.badRequest(
```

Comment:
resolving a runtime error for input validations brought about by the recent refactoring works.